### ISSUE TITLE
Module 3 dynamics layer colours

### ISSUE THREAD

**Issue by thejollygeo** [Blue-Ventures-Conservation/GEM#53]
As discussed on our call last week, could you switch the persistence and gain colours around

Currently we have:
loss = red
persistence = blue
gain = green

<img width="700" alt="Image" src="https://github.com/user-attachments/assets/017d6f10-6cf8-4ac8-9e20-26e82b26b081" />

Could we change to:
loss = red
persistence = green
gain = blue

could you also make the blue ever so slightly darker, and the green ever so slightly brighter?  So that the gain pixels stand out a little more

Assigned 'high' as need for the Somone slides.  Thanks!
